macros/apply-tags/unapply_mps_from_tags.sql

```diff
@@ -7,20 +7,25 @@
 
   {% set ns = ns or dbt_tags.get_resource_ns() %}
   {% set tags = dbt_tags.get_adapter_tags(ns=ns) %}
+  {% set column_tags = [] %}
```

Comment:
I have this error trying to run the macro. Not sure where it comes from
![image](https://github.com/user-attachments/assets/586e02ec-0bcf-43c8-8ec8-8e4f58e39fec)

Reply:
@thu-IL Can you git pull and try again?

Reply:
It works now. Thank you